### PR TITLE
Added java code snippet for hyperloglog

### PR DESCRIPTION
**Pull request by Ranjeet0611**
Added java code snippet for hyperloglog #3629

### CODE REVIEW

**Comment by Ranjeet0611** [comment]
Hi @dwdougherty Please review this example
Thank you

**Review by dwdougherty — APPROVED** [review]
LGTM. Thanks again, Ranjeet!

**Review by sazzad16 — COMMENTED** [review]
@dwdougherty Weren't the `import`s supposed to be removed or at least hidden?

@Ranjeet0611 Here is only one object of UnifiedJedis. So use a shorter variable name; like jedis, client, etc.

**Comment by Ranjeet0611** [comment]
@sazzad16 did the changes
Thank you

**Comment by dwdougherty** [comment]
> @dwdougherty Weren't the `import`s supposed to be removed or at least hidden?

Only the code between `// STEP_START` and `// STEP_END` is shown in the doc code tabs, so hiding imports, especially this small number of them, is optional.

File: src/test/java/io/redis/examples/HyperLogLogExample.java [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+        System.out.println(res1); // >>> 1
+
+        long res2 = jedis.pfcount("bikes");
+        System.out.println(res2); // >>> 4
+
+        long res3 = jedis.pfadd("commuter_bikes", "Salacia", "Mimas", "Quaoar");
+        System.out.println(res3); // >>> 1
+
+        String res4 = jedis.pfmerge("all_bikes", "bikes", "commuter_bikes");
+        System.out.println(res4); // >>> True
```
Comment (sazzad16) [inline src/test/java/io/redis/examples/HyperLogLogExample.java:30]: The reply of a successful PFMERGE command is an String `OK`.
Please run your code and write output properly.

**Review by sazzad16 — CHANGES_REQUESTED** [review]

File: src/test/java/io/redis/examples/HyperLogLogExample.java (hunk 1 above)
Comment (Ranjeet0611) [reply src/test/java/io/redis/examples/HyperLogLogExample.java:30]: Sure

**Comment by Ranjeet0611** [comment]
@dwdougherty  please review

**Review by dwdougherty — APPROVED** [review]
@sazzad16 This PR builds correctly and it's good to go.

**Review by sazzad16 — APPROVED** [review]

**Pull request merged**